package.json

```diff
@@ -23,6 +23,8 @@
   "files": ["README.md", "LICENSE.md", "package.json", "priv/static/phoenix.js", "assets/js/phoenix.js"],
   "scripts": {
     "test": "./node_modules/.bin/mocha ./assets/test/**/*.js --compilers js:babel-register -r jsdom-global/register",
-    "docs": "documentation build assets/js/phoenix.js -f html -o doc/js"
+    "docs": "documentation build assets/js/phoenix.js -f html -o doc/js",
+    "watch": "brunch watch",
+    "build": "brunch build"
```

Comment:
The `./node_modules/.bin` directory is added to the `$PATH` by npm when it runs scripts, so just `brunch watch` and such is fine.